### PR TITLE
fix UnicodeEncodeError when writing to the changelog

### PR DESCRIPTION
**Pull request by szymonlopaciuk**
Replaced open with codecs.open to write unicode to the `CHANGELOG.txt` file as UTF-8.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
 from __future__ import print_function
 
 import os
+import codecs
```
Comment (jacquerie) [inline docs/conf.py]: You should be able to do `from io import open` here and then you won't have to touch the code below. `codecs` only matters when you have to support Python 2.5 (which is never).

Comment (szymonlopaciuk) [reply docs/conf.py]: Done in d7cd775

**Comment by michamos** [comment]
can you rebase on master? I just merged #261.

File: docs/conf.py (hunk 1 above)
Comment (jacquerie) [reply docs/conf.py]: > Done in d7cd775

Something went wrong: this commit still contains the old code.

Comment (jacquerie) [reply docs/conf.py]: Ok, you pushed be2a5fc to fix this.

Comment (szymonlopaciuk) [reply docs/conf.py]: yup, messed up. Should be fine now

**Comment by szymonlopaciuk** [comment]
@michamos rebased 👍

**Pull request merged**